language/move-prover/bytecode-to-boogie/src/spec_translator.rs

```diff
@@ -22,6 +22,7 @@ use crate::env::{
     FunctionEnv, GlobalType, ModuleEnv, ModuleIndex, StructEnv, TypeParameter, ERROR_TYPE,
     UNKNOWN_TYPE,
 };
+use pretty::DocAllocator;
```

Comment:
Unnecessary import? You should get a warning from rustc. There must be no warnings for the code.

Reply:
I removed it.